.secrets.baseline

```diff
@@ -77,7 +77,7 @@
     }
   ],
   "results": {},
-  "version": "0.13.1+ibm.61.dss",
+  "version": "0.13.1+ibm.60.dss",
```

Comment:
this is an older version than what is in source control. You must have an outdated version of common-dev-assets locally. Please update